.github/workflows/end-to-end-tests.yaml

```diff
@@ -40,17 +40,16 @@ jobs:
             test/end-to-end-tests/synapse/installations/consent/homeserver.log
             retention-days: 14
 
-      - name: Download previous benchmark data
```

Comment:
This is a dead block, as we aren't using the "separate file" version of this benchmark action.